homeassistant/components/palazzetti/strings.json

```
@@ -47,6 +47,35 @@
           }
         }
       }
+    },
+    "sensor": {
+      "pellet_quantity": {
+        "name": "Pellet quantity"
+      },
+      "pellet_level": {
+        "name": "Pellet level"
+      },
+      "air_outlet_temperature": {
+        "name": "Air outlet temperature"
+      },
+      "wood_combustion_temperature": {
+        "name": "Wood combustion temperature"
+      },
+      "room_temperature": {
+        "name": "Room temperature"
+      },
+      "return_wate_temperature": {
```

```suggestion
      "return_water_temperature": {
```
I mean, we do have a typo here. Setting them all up would ensure they always get their correct name and value returned. This would probably only mean you have to extend the `list_temperatures` mock, so personally I'd go for it.

---

Darn, excellent point, I was sure I had fixed that typo already :/ 

Only one questionwith your suggestion: I can definitely add more temperature probes to the mock, but it won't resemble any existing franken-device. Are we fine with that?

---

Thanks. I found a bug when I added the sensors to the snapshot test, so I'm glad that you called that out.